edb/server/main.py

```diff
@@ -487,13 +510,30 @@ def _protocol_version(
 
 
 def _validate_max_backend_connections(ctx, param, value):
-    if value < defines.BACKEND_CONNECTIONS_MIN:
+    if value is not None and value < defines.BACKEND_CONNECTIONS_MIN:
         raise click.BadParameter(
             f'the minimum number of backend connections '
             f'is {defines.BACKEND_CONNECTIONS_MIN}')
     return value
 
 
+def _default_max_backend_connections():
+    try:
+        import psutil
```

Comment:
We don't have `psutil` as a dependency in our `setup.py`, and since we ship our own Python in our packages it's a 0 chance that this import will ever succeed on a user installation.

I think we can add psutil as a dependency, this isn't the first time we need it. @elprans ok?

Reply:
Added now 🖖